### ISSUE TITLE
Kekulize error in version 2022.09

### ISSUE THREAD

**Issue by Sunghun-Bae** [rdkit/rdkit#5719]
**Describe the bug**
Following code works for rdkit 2021.03.1 and 2022.03.5 but failed in 2022.09

**To Reproduce**
  1 import rdkit
  2 from rdkit import Chem
  3 from rdkit.Chem.MolStandardize import rdMolStandardize
  4
  5 print(rdkit.__version__)
  6
  7 ini_smi = 'CC1=CC(NC(=O)C(F)(F)F)=NC(C)=N1'
  8
  9 m = Chem.MolFromSmiles(ini_smi)
 10 enumerator = rdMolStandardize.TautomerEnumerator()
 11 can_mol = enumerator.Canonicalize(m)
 12 can_smi = Chem.MolToSmiles(can_mol)
 13
 14 print(ini_smi)
 15 print(can_smi)

**Expected behavior**
2021.03.1
CC1=CC(NC(=O)C(F)(F)F)=NC(C)=N1
Cc1cc(NC(=O)C(F)(F)F)nc(C)n1

**Screenshots**
2022.09.1
[15:57:21] Can't kekulize mol.  Unkekulized atoms: 2 12
Traceback (most recent call last):
  File "/home/shbae/test-rdkit-2022-09-bug.py", line 10, in <module>
    can_mol = enumerator.Canonicalize(m)
rdkit.Chem.rdchem.KekulizeException: Can't kekulize mol.  Unkekulized atoms: 2 12
(rdkit) python test-rdkit-2022-09-bug.py
2022.09.1
[15:58:24] Can't kekulize mol.  Unkekulized atoms: 2 12
Traceback (most recent call last):
  File "/home/shbae/test-rdkit-2022-09-bug.py", line 11, in <module>
    can_mol = enumerator.Canonicalize(m)
rdkit.Chem.rdchem.KekulizeException: Can't kekulize mol.  Unkekulized atoms: 2 12

**Configuration (please complete the following information):**
 - RDKit version: 2022.09.1

**Comment by greglandrum** [comment]
Confirmed. I can reproduce this on master.

**Comment by eguidotti** [comment]
Same here

**Comment by brianlahue** [comment]
I've seen the same.  On the surface, the problem seems to be related to the substituents:

```
enumerator.Canonicalize(Chem.MolFromSmiles("Cc1cc(nc(n1)C)N"))  # amine, 2,6-dimethyl --> errors!
enumerator.Canonicalize(Chem.MolFromSmiles("Cc1cc(ncn1)N"))  # no 2-methyl -->  errors!
enumerator.Canonicalize(Chem.MolFromSmiles("Cc1nccc(n1)N"))  # no 6-methyl -->  works!
enumerator.Canonicalize(Chem.MolFromSmiles("Cc1ccnc(n1)C"))  # no amine -->  works!
enumerator.Canonicalize(Chem.MolFromSmiles("Cc1cc(nc(n1)C)C"))  # replace amine with methyl --> works!
```

**Comment by greglandrum** [comment]
This should have been fixed by the fix for #5784

**Comment by linzihan322** [comment]
@greglandrum It can still be reproduced in 2024_03_6
